### ISSUE TITLE
Accessibility: Fix cover drag and drop on mobile

### ISSUE THREAD

**Issue by LeadSongDog** [internetarchive/openlibrary#905]
The cover drag and drop code around:
https://github.com/internetarchive/openlibrary/blob/d060d97e1da814bac97c74771933f6ab3c0c8829/openlibrary/templates/covers/manage.html#L25
does not work on mobile. 
See https://github.com/timruffles/mobile-drag-drop for more info.

For accessibility, there should be a non-drag option as well: one click to select, then cut and paste.

**Comment by guyjeangilles** [comment]
What's the use case for dragging and dropping on mobile? I'm not convinced that this will be useful.

I'm assigning jdlrobson (not tagging as request) per slack discussion since this seems javascript/front-end related

**Comment by LeadSongDog** [comment]
@guyjeangilles The UI only supports one way to delete a cover: drag and drop. Since that does not work on any iOS device, nor is it really accessible on other systems, a simpler alternative is still needed.

**Comment by seabelis** [comment]
I've gotten several patron requests for this as well.

**Comment by cdrini** [comment]
Bumping up in priority due to demand.
